hokusai/production.yml

```diff
@@ -1,12 +1,107 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: force
+    layer: application
+    component: web
+    spot-allowed: false
+    app.kubernetes.io/version: production
+  name: force-web1
+spec:
+  replicas: 3
```

Comment:
This^ would be the bare minimum scale during spot disruptions?

Reply:
yes. there's a trade-off here. higher replica count means less degradation but also less Spot utilization. lower count means higher Spot utilization but more degradation.